continuous-integration/check.sh

```diff
@@ -0,0 +1,49 @@
+#!/bin/bash -e
+
+# Get and install dependencies
+echo
+echo '-- Prepare --'
+echo "Running on: $DISTRO $OS"
+cd /cwd
+
+if [[ $DISTRO == "ubuntu" ]]; then
+    apt-get update -qq
+    apt-get -y install build-essential
+    apt-get -y install libmount-dev libpcre3-dev python-dev dh-autoreconf flex bison pkg-config zlib1g-dev libffi-dev
+    apt-get -y install libxml2-utils libgamin-dev
+    apt-get -y install software-properties-common
+    add-apt-repository -y ppa:ricotz/testing
+    apt-get update
+    apt-get -y install libmozjs-31-dev
+else
+    exit 1
+fi
+
+# Build glib 2.0
```

Comment:
Have you considered using JHbuild instead of doing this? We could add a `.jhbuildrc` file that pins all dependencies to a particular version so that we don't get build failures from bugs in GLib, etc.

It might not be the right solution in the end, but I'd be interested to hear your preference.

JHbuild would allow for a much simpler and distro-independent way to install dependencies, because you could just do `jhbuild sysdeps gjs --install` and get them automatically (e.g. using `apt-file` on Ubuntu or Debian.)